plugins/hls-explicit-imports-plugin/test/Main.hs

```diff
@@ -34,10 +34,43 @@ main = defaultTestRunner $ testGroup "import-actions"
     "Make imports explicit"
     [ codeActionAllGoldenTest "ExplicitUsualCase" 3 0
     , codeActionAllResolveGoldenTest "ExplicitUsualCase" 3 0
+    , inlayHintsTest "ExplicitUsualCase" 2 $ (@=?)
+        [InlayHint
+           { _position = Position {_line = 2, _character = 16}
+           , _label = InL "( a1 )"
+           , _kind = Just InlayHintKind_Type
+           , _textEdits = Just [TextEdit (Range (Position 2 0) (Position 2 16)) "import ExplicitA ( a1 )"]
+           , _tooltip = Nothing
+           , _paddingLeft = Just True
+           , _paddingRight = Nothing
+           , _data_ = Nothing
+           }]
     , codeActionOnlyGoldenTest "ExplicitOnlyThis" 3 0
     , codeActionOnlyResolveGoldenTest "ExplicitOnlyThis" 3 0
+    , inlayHintsTest "ExplicitOnlyThis" 3 $ (@=?)
```

Comment:
We should test that we _don't_ get code actions when we have inlay hints enabled and vice versa.